packages/cozy-stack-client/src/__tests__/setup.js

```diff
@@ -35,6 +35,17 @@ expect.extend({
   }
 })
 
+beforeEach(() => {
+  jest.spyOn(global.console, 'log').mockImplementation(() => {})
+  jest.spyOn(global.console, 'info').mockImplementation(() => {})
+})
```

Comment:
What is the reason behind this change?

Reply:
So many console.log polluting the test output. It's better to display them on a test by test basis, with a `console.log.mockRestore()`.

Reply:
Meh, ok, but we should eventually review why we are writing so much to the console :)

Reply:
When I write to the console, it is to have useful debugging imformation. It is wrapped in a `NODE_ENV !== 'production'`.